src/Bundler/Runner/js/tsc.js

```diff
@@ -5,9 +5,11 @@ function compile(source) {
         compilerOptions: {
             inlineSourceMap: false,
             skipLibCheck: true,
+            importHelpers: true,
             target: ts.ScriptTarget.ES5,
             module: ts.ModuleKind.CommonJS,
             moduleResolution: ts.ModuleResolutionKind.NodeJs,
+            emitDecoratorMetadata: true,
```

Comment:
Why would we enable this?

The docs of typescript point to this issue: https://github.com/Microsoft/TypeScript/issues/2577

But that does not tell me anything about why this is (or isn't) a good idea.

Reply:
Angular require this to be turned on.